src/utils/export-map.ts

```diff
@@ -1117,6 +1102,49 @@ function childContext(
   }
 }
 
+const optionsHashesCache: Record<
+  string,
+  { value: string; hash: string } | undefined
+> = {}
```

Comment:
If it is a cache, maybe `Map` should be used here.

Reply:
On the one hand yes, it's better to use `Map` for any cache. However, this object has three known properties and using `Map` here seems a bit overkill

Maybe it's better to just declare all known properties immediately when creating the object and don't use `Map` at all?

Reply:
> Maybe it's better to just declare all known properties immediately when creating the object

If it is possible, please do so! The monomorphic approach is the most perfomant.